include/maxscale/service.h

```diff
@@ -65,6 +65,7 @@ typedef struct server_ref_t
     double               server_weight; /**< Weight in the range [0..1]. 0 is worst, and a special case. */
     int                  connections;   /**< Number of connections created through this reference */
     bool                 active;        /**< Whether this reference is valid and in use*/
+    uint64_t              lru_clock;
```

Comment:
This could be declared inside the function as a static variable to reduce the scope.